js/sondehub.js

```diff
@@ -858,20 +858,35 @@ function clean_refresh(text, force, history_step) {
 }
 
 function load() {
+    var qm = offline.get("qm");
+    if (qm != null) {
+        wvar.mode = qm;
+    }
+
+    coords = offline.get("mc");
+    if (coords == null) {
```

Comment:
In theory the map should use browser location if possible - but I don't think thats working; any chance you want to have a look?

Reply:
I don't think anything should have changed with that. When browser coordinates are available, they are displayed in the upper left, and clicking on them moves the map center.